src/libaudqt/info-widget.cc

```diff
@@ -116,6 +127,34 @@ EXPORT bool InfoWidget::updateFile ()
     return m_model->updateFile ();
 }
 
+void InfoWidget::toClipboard ()
+{
+    QAction *action = qobject_cast<QAction*>(sender ());
+    if (action)
+    {
+        QModelIndex index = action->data().toModelIndex ();
+        QMimeData *data = new QMimeData;
+        data->setText (m_model->data (index, Qt::DisplayRole).toString());
+        QApplication::clipboard ()->setMimeData (data);
+    }
+}
+
+void InfoWidget::contextMenu (const QPoint & pos)
+{
+    QModelIndex index = indexAt (pos);
+    if (index.column () == 1)
```

Comment:
This would be a really good time to add an enum for the columns, rather than hard-coding a "1" here.